lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -1,20 +1,26 @@
 
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
 import 'package:core/utils/app_logger.dart';
 import 'package:core/utils/build_utils.dart';
+import 'package:dartz/dartz.dart';
 import 'package:enough_html_editor/enough_html_editor.dart';
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/services.dart';
 import 'package:get/get.dart';
 import 'package:html_editor_enhanced/html_editor.dart';
 import 'package:jmap_dart_client/jmap/account_id.dart';
 import 'package:jmap_dart_client/jmap/core/id.dart';
 import 'package:jmap_dart_client/jmap/core/properties/properties.dart';
+import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
 import 'package:jmap_dart_client/jmap/identities/identity.dart';
 import 'package:jmap_dart_client/jmap/mail/email/email_address.dart';
-import 'package:model/extensions/email_address_extension.dart';
-import 'package:model/extensions/identity_extension.dart';
-import 'package:model/user/user_profile.dart';
+import 'package:model/model.dart';
```

Comment:
avoid `import 'package:model/model.dart';`
need use 
```suggestion
avoid `import 'package:model/xxxobject.dart';`
```

Reply:
model include many things, please only import what you need?